api/v1alpha1/project_config_types.go

```diff
@@ -114,9 +119,24 @@ type GitHubWebhookReceiverConfig struct {
 	// the shared secret used to authenticate the webhook requests sent by GitHub.
 	// For more information please refer to GitHub documentation:
 	//   https://docs.github.com/en/webhooks/using-webhooks/validating-webhook-deliveries
+	SecretRef corev1.LocalObjectReference `json:"secretRef" protobuf:"bytes,1,opt,name=secretRef"`
+}
+
+// BitbucketWebhookReceiverConfig describes a webhook receiver that is
+// compatible with Bitbucket payloads.
+type BitbucketWebhookReceiverConfig struct {
+	// SecretRef contains a reference to a Secret. For Project-scoped webhook
+	// receivers, the referenced Secret must be in the same namespace as the
+	// ProjectConfig.
+	//
+	// For cluster-scoped webhook receivers, the referenced Secret must be in the
+	// designated "cluster Secrets" namespace.
 	//
-	// The value of the token key goes in the "Secret" field when registering a
-	// GitHub App or webhook in the GitHub UI.
+	// The Secret's data map is expected to contain a `bitbucket-secret` key whose
```

Comment:
![Screenshot 2025-06-10 at 11 04 44 PM](https://github.com/user-attachments/assets/8b9b6670-2860-4bbd-b93e-704c7ed8d93a)

The registration form does call this field `secret`, but I don't believe prefixing that with `bitbucket-` adds any value here. In fact, it looks strange.

@nitishfy let's please change this to just `secret`.

Codegen will need to be re-run.